src/Behat/Testwork/Exception/ExceptionPresenter.php

```diff
@@ -93,13 +94,45 @@ public function presentException(Exception $exception, $verbosity = null)
             }
         }
 
-        if (OutputPrinter::VERBOSITY_VERY_VERBOSE <= $verbosity) {
-            return $this->relativizePaths(trim($exception));
+        if (OutputInterface::VERBOSITY_VERY_VERBOSE <= $verbosity) {
+            if (OutputInterface::VERBOSITY_DEBUG > $verbosity) {
```

Comment:
the old `OutputPrinter` verbosity constants were still used. These are deprecated and the ones from Symfony's `OutputInterface` are used throughout Behat. As their values are not equal, we should change them here as well.